src/components/Modal/Modal.jsx

```diff
@@ -53,4 +54,12 @@ const Modal = ({ isModalActive, type, title, content, onModalClose }) => {
     );
 };
 
+Modal.propTypes = {
+    isModalActive: PropTypes.bool.isRequired,
+    type: PropTypes.oneOf(['information', 'confirmation']),
```

Comment:
Can you please create file with const modalTypes contains two fields with strings values?